apps/homepage/src/app/[lang]/page.tsx

```diff
@@ -1,13 +1,57 @@
 'use client'
 import styles from './home.module.css'
-import { useLocalization } from '../../lib/context/loc-context'
+import { motion, AnimatePresence } from 'framer-motion';
+import { useEffect, useState } from 'react';
+
+const keyWords = [
```

Comment:
What about the rest of the phrase?

Reply:
Let's change the approach to accept like a template string with the values that will be animated in the key parts. I was thinking something like:

```
  This is my {keyWord}
```

and then in an object something like:

```
const phraseData = {
  phrase: "This is my {keyWord}",
  keyWords: [
    "word 1",
    "word 2"
  ]
}
```

This approach makes the phrase and the key word dinamic, the only issue is this suggestion only handles 1 key word, we should have an approach where we have `n` number of keywords.

Reply:
You could start by creating this structure without the animation for now and once you can render the phase you can focus on the animation.

Reply:
This should also work:
```
const phraseData = { 
  phrase: "This is my {keyWord} and I can change it for another {keyWord} and another {keyWord}", 
  keyWords: .....
}
```